recipes/spark_historyserver.rb

```diff
@@ -34,11 +34,11 @@
   if node['spark']['spark_defaults'].key?('spark.eventLog.dir')
     node['spark']['spark_defaults']['spark.eventLog.dir']
   else
-    '/user/spark/applicationHistory'
+    'hdfs:///user/spark/applicationHistory'
```

Comment:
I've tried to replicate the Spark defaults correctly. Unfortunately, their defaults are effing stupid. They set this directory to `/user/spark/applicationHistory` whether we're in HDFS or on a local file-system. Basically, I'm trying to do the correct thing here, and there's not an easy way to do it because the defaults are incorrect and I have no way of knowing ahead of time if this is an HDFS cluster or if this is a standalone Spark cluster with no HDFS.